packages/frontmatter/src/FrontmatterBlock.tsx

```diff
@@ -74,11 +74,20 @@ export function DateString({
   spacer?: boolean;
 }) {
   if (!date) return null;
-  const d = new Date(date); // This is in the users timezone
-  const utcDate = new Date(d.getUTCFullYear(), d.getUTCMonth(), d.getUTCDate());
+  // Parse the date, either using date's intrinsic timezone, or local timezone
+  const d = new Date(date);
+  // Rebuild the timezone aware date in UTC without applying the TZ shift
+  const utcDate = new Date(Date.UTC(d.getFullYear(), d.getMonth(), d.getDate()));
```

Comment:
We might not even need to do this now; https://github.com/jupyter-book/mystmd/pull/1426 ensures that the validated date has `dddd-mm-yy` format, so we could just parse it as a string and then build `utcDate` from the components.

Reply:
we should just pass yyyy-mm-dd into the `<time>` element https://developer.mozilla.org/en-US/docs/Web/HTML/Element/time it supports it.